### PR TITLE
Remove all direct references to font-awesome

### PR DESCRIPTION
**Pull request by yardenshoham**
- Related #10410
- I had to add an SVG for an empty checkbox https://github.com/primer/octicons/issues/942

# Before
image

# After
image

### CODE REVIEW

**Comment by yardenshoham** [comment]
(following @delvh's update yesterday)

**Comment by delvh** [comment]
How the hell did you notice my update there yesterday?
Message edits don't trigger a notification, right?
Or is it simply a coincidence that you opened this PR the day after?

**Comment by yardenshoham** [comment]
~I'm everywhere~ sort by recently updated

File: web_src/js/svg.js [hunk 1]
```diff
@@ -91,6 +93,8 @@ const svgs = {
   'octicon-tag': octiconTag,
   'octicon-git-branch': octiconGitBranch,
   'octicon-rss': octiconRss,
+  'octicon-checkbox': octiconCheckbox,
+  'gitea-empty-checkbox': giteaEmptyCheckbox,
```
Comment (delvh) [inline web_src/js/svg.js]: Oh, I see that the alphabetic sorting of this list has been broken.
Would you mind to restore it?

**Review by delvh — APPROVED** [review]

File: web_src/js/svg.js (hunk 1 above)
Comment (yardenshoham) [reply web_src/js/svg.js]: done (sorted by the second word)

Comment (delvh) [reply web_src/js/svg.js]: Thanks :+1:

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Can't be much left now for fontawesome removal.

**Pull request merged**